samples/java-spring-cinema-booking-choreography/src/it/java/com/example/cinema/ShowCalls.java

```diff
@@ -0,0 +1,77 @@
+package com.example.cinema;
+
+import com.example.cinema.model.CinemaApiModel;
+import com.example.cinema.model.Show;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+
+
+@Component
+public class ShowCalls {
```

Comment:
We should remove this and use ComponentClient. No need to build on by hand.

I think that originally, when the blog post were written, we didn't have the ComponentClient yet.

Reply:
I agree. We did have ComponentClient then, it is used in other places